examples/storage/cassandra/README.md

```diff
@@ -165,18 +165,18 @@ cassandra   None         <none>        9042/TCP   45s
 
 If an error is returned the service create failed.
 
-## Step 2: Use a Pet Set to create Cassandra Ring
+## Step 2: Use a StatefulSet to create Cassandra Ring
```

Comment:
Also is it StatefulSet or Stateful Set?  The object is called StatefulSet, but in English how are we defining it?  Again please mention that this feature was previously named PetSet.